doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,785 @@
+for the messagelog database) and has some implications on the security server resource requirements as a separate
+instance uses some memory.
+
+
+###### 2.3.2.2 OCSP responses from `/var/cache/xroad/`: not replicated
+
+The OCSP responses are currently not replicated. Replicating them could make the cluster more fault tolerant but the
+replication cannot simultaneously create a single point of failure. A distributed cache could be used for the responses.
+
+
+## 3. X-Road Installation and configuration
+
+You can set up the cluster manually, or use the provided Ansible playbook \[[SS-CLUSTER](#12-references)\] if it suits
+your purposes.
+
+### 3.1 Prerequisites
+
+In order to properly set up the data replication, the slave nodes must be able to connect to:
+* the master server using SSH (tcp port 22), and
+* the master `serverconf` database (e.g. tcp port 5433).
+
+
+### 3.2 Master installation
+
+1. Install the X-Road security server packages using the normal installation procedure or use an existing standalone node.
+2. Stop the xroad services.
+3. Create a separate PostgreSQL instance for the `serverconf` database (see section
+   [4. Database replication setup](#4-database-replication-setup) for details).
+   * Change `/etc/db.properties` to point to the separate database instance
+4. If you are using an already configured server as the master, the existing configuration was replicated to the slaves
+   in step 3. Otherwise, proceed to configure the master server: install the configuration anchor, set up basic information,
+   create authentication and signing keys and so on. See the security server installation guide \[[IG-SS](#12-references)\]
+   for help with the basic setup.
+5. Set up the configuration file replication, see section
+   [5. Configuring data replication with rsync over SSH](#5-configuring-data-replication-with-rsync-over-ssh)
+   * Additionally, `rssh` shell can be used to to restrict slave access further, but note that it is not available on RHEL.
+
+7. Configure the node type as `master` in `/etc/xroad/node.ini`:
+      ```bash
+      [Node]
+      type=master
+      ```
+8. Start the X-Road services.
+
+
+### 3.3 Slave installation
+1. Install security server packages using the normal installation procedure. `nginx` or `xroad-jetty` packages  are not
+   required for slave nodes, but the admin graphical user interface (which requires these packages) can be handy for
+   diagnostics. It should be noted that changing a slave's configuration via the admin gui is not possible.
+2. Stop the xroad services.
+3. Create a separate PostgreSQL instance for the serverconf database (see section
+   [4. Database replication setup](#4-database-replication-setup) for details)
+  * Change `/etc/db.properties` to point to the separate database instance and change password to match the one defined in the master database.
```

Comment:
It was a little bit confusing to follow the configuration process when it is divided in so many separate chapters, and some chapters are specific to master or slave, and some talk about both. Here we start from chapter 3.2.3 -> jump to chapter 4 -> finish 4, should go back to 3.2.3 (and not carry on to 5) -> continue to 3.2.5. -> jump to 5 -> come back to 3.3, continue with slave installation and notice you have already done most of the slave stuff in chapters 4 & 5